src/components/NavigationPanel.jsx

```diff
@@ -31,83 +33,105 @@ function NavigationPanel() {
       </div>
       <ul
         className={`
-         flex flex-col sm:justify-center sm:gap-10 pl-4 pr-4
-         w-1/3 md:w-full absolute sm:relative bg-white px-4 h-screen 
-         ${!isNavOpen && 'sm:flex'}
+         flex flex-col sm:justify-center sm:gap-6
+         px-2
+         overflow-hidden
+         md:w-full absolute sm:relative bg-white h-screen 
+         ${!isNavOpen && "sm:flex"}
          z-40
          transition-all duration-500 ease-in-out
          sm:left-0
-         transform ${isNavOpen ? 'left-0' : 'left-[-100%]'}
+         transform ${isNavOpen ? "left-0" : "left-[-100%]"}
+         font-bold text-xl sm:text-md uppercase text-gray-600
        `}
       >
-        <li>
-          <Link to="/" className="hover:bg-[#94bc0c] pl-4 py-4 w-full block">
-            Home
-          </Link>
-        </li>
-        {isAuthenticated ? (
-          <>
-            <li>
-              <Link
-                to="/reserve"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Reserve
-              </Link>
-            </li>
-            <li>
-              <Link
-                to="/my-reservations"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Reservations
-              </Link>
-            </li>
-            <li>
-              <Link
-                to="/add-concert"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Add Concert
-              </Link>
-            </li>
-            <li>
-              <Link
-                to="/delete-concert"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Delete Concert
-              </Link>
-            </li>
-            <li>
-              <button
-                onClick={handleLogout}
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block text-left"
-              >
-                Logout
-              </button>
-            </li>
-          </>
-        ) : (
-          <>
-            <li>
-              <Link
-                to="/login"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Login
-              </Link>
-            </li>
-            <li>
-              <Link
-                to="/signup"
-                className="hover:bg-[#94bc0c] pl-4 py-4 w-full block"
-              >
-                Signup
-              </Link>
-            </li>
-          </>
-        )}
+        <a href="http://localhost:5173" className="site-logo absolute top-10">
+          <img src={Logo} alt="nav-logo" className="w-64 h-fit" />
+        </a>
```

Comment:
@SandipGyawali Nice Job including the logo here, however, submitting this code to production would make  this link unable to work, please consider using relative links like in the example below:
![image](https://github.com/Diegogagan2587/Book-a-concert-front-end/assets/61764778/d61980e1-b6bd-472d-822c-844ce30114a1)
The previous example makes us of the router and helps to implement the relative link which would work here and in production as expected.

Reply:
ok for sure i will fix it

Reply:
@Diegogagan2587 can't we use typescript instead of javascript it might be good if we add type saftey which will help in writing quality code and also maintainable. What's your opinion

Reply:
@SandipGyawali TypeScript is an excellent choice that might improve clarity on some components, and increase confidence in our code, is worth considering using it, however, it might break packages and we might have to re-configure the workflow. I would prefer to use typescript on a new and different project instead of refactoring this one. Any ways, thanks for asking!

Reply:
@Diegogagan2587 ok cool, and for the ui issue that we faced it's been fixed if there is any error or missout mention it to me

Reply:
@danifromecuador I'd appreciate it if you could review these changes whenever you have time.

Reply:
@danifromecuador can you check the request ?